docs/guides/configuration.md

```diff
@@ -60,6 +60,27 @@ npx puppeteer browsers install
 
 ### Examples
 
+#### Downloading multiple browsers
+
+Starting in v23.0.0, Puppeteer allows you to download multiple browser
+without the need run multiple commands:
+
+```js title="project-directory/.puppeteerrc.cjs"
+/**
+ * @type {import("puppeteer").Configuration}
+ */
+module.exports = {
+  // Explicitly confirm downloading Chrome
+  chrome: {
+    skipDownload: false,
+  },
+  // Change default behavior to downloading Firefox
+  firefox: {
+    skipDownload: false,
+  },
+};
+```
+
```

Comment:
let's add that after changing the puppeteer config, you need to run `npx browsers install` to download all configured browsers